src/_data/side-nav.yml

```diff
@@ -138,6 +119,8 @@
     - title: dart:html
       permalink: /libraries/dart-html
     - divider
+    - title: Iterable collections
+      permalink: /codelabs/iterables
```

Comment:
**suggestion:** Move this into the language section and maybe have it not include ToC entry. The user can find it from the Collections page.

Also, change the URL to `language/collections/iterables` or `language/iterables`.

Reply:
This page goes into the `Iterable` type and its members, so it seems to me a library topic rather than about a language feature. 

Besides that, we'd like to eventually replace this content anyway, so it likely doesn't belong in the more standardized language section.